docs/api-ref/rest-api/index.md

```diff
@@ -11,13 +11,16 @@ tags:
 
 # REST API
 
-The [Prefect REST API](/api-ref/rest-api/) is used for communicating data from clients to the Prefect server so that orchestration can be performed. This API is consumed by clients such as the Prefect Python SDK or the server dashboard.
+The [Prefect REST API](https://deploy-preview-9661--prefect-docs-preview.netlify.app/api-ref/rest-api/) is used for communicating data from clients to the Prefect server so that orchestration can be performed. This API is consumed by clients such as the Prefect Python SDK or the server dashboard.
```

Comment:
```suggestion
The [Prefect REST API](/api-ref/rest-api/) is used for communicating data from clients to the Prefect server so that orchestration can be performed. This API is consumed by clients such as the Prefect Python SDK or the server dashboard.
```

Reply:
why did they all get changed to deploy preview?

Reply:
I believe I copy-pasted your suggested change. Will fix those. Good catch!